index.js

```diff
@@ -0,0 +1,83 @@
+let middleDiv = document.querySelector("#middleDiv");
+let selectList = document.querySelector("#selectList");
+let allMoviesData;
+
+document.addEventListener("DOMContentLoaded", () => {
+    console.log("DOM loaded");
+    fetchMovies();
+    document.getElementById("selectList").addEventListener("change", function() {
+        resultsFromOption(allMoviesData);
+    });
+
+    document.getElementById('my-form').onsubmit = function() {
+        return isValidForm();
+    };
+    let form = document.querySelector("#my-form");
+    let ul = document.createElement("ul");
+    ul.id = "ul";
+    form.appendChild(ul);
+    })
+
+const fetchMovies = () => {
+    console.log("fetchMovies started");
+    fetch("https://ghibliapi.herokuapp.com/films")
+        .then(response => {
+            console.log(response);
+            return response.json();
+        })
+        .then(dataObj => {
+            console.log("dataObj: ", dataObj);
+            allMoviesData = dataObj;
+            renderMovies(dataObj);
+        })
+        .catch(err => {
+            console.log("err: ", err);
+        })
+    console.log("fetchMovies finished");
+}
+
+const renderMovies = (movies) => {
+    let selectList = document.querySelector("#selectList");
+    let id = 0;
+    for(let i = 0; i < movies.length; i++) {
+        let movieOption = document.createElement("option");
+        movieOption.value = movies[i].title;
+        movieOption.text = movies[i].title;
+        movieOption.id = `movie${id}`;
```

Comment:
You could probably just use "i" value for id since i and id does exactly the same thing.